pyweather.py

```diff
@@ -1909,7 +1956,156 @@ def radar_clearImages():
+    yesterday_maxTempF = str(data['maxtempi'])
+    yesterday_maxTempC = str(data['maxtempm'])
+    yesterday_minTempF = str(data['mintempi'])
+    yesterday_minTempC = str(data['mintempm'])
+    logger.debug("yesterday_maxTempF: %s ; yesterday_maxTempC: %s" %
+                 (yesterday_maxTempF, yesterday_maxTempC))
+    logger.debug("yesterday_minTempF: %s ; yesterday_minTempC: %s" %
+                 (yesterday_minTempF, yesterday_minTempC))
+    yesterday_maxDewpointF = str(data['maxdewpti'])
+    yesterday_maxDewpointC = str(data['maxdewptm'])
+    yesterday_minDewpointF = str(data['mindewpti'])
+    yesterday_minDewpointC = str(data['mindewptm'])
+    logger.debug("yesterday_maxDewpointF: %s ; yesterday_maxDewpointC: %s" %
+                 (yesterday_maxDewpointF, yesterday_maxDewpointC))
+    logger.debug("yesterday_minDewpointF: %s ; yesterday_minDewpointC: %s" %
+                 (yesterday_minDewpointF, yesterday_minDewpointC))
+    yesterday_maxPressureInHg = str(data['maxpressurei'])
+    yesterday_maxPressureMB = str(data['maxpressurem'])
+    yesterday_minPressureInHg = str(data['minpressurei'])
+    yesterday_minPressureMB = str(data['minpressurem'])
+    logger.debug("yesterday_maxPressureInHg: %s ; yesterday_maxPressureMB: %s" %
+                 (yesterday_maxPressureInHg, yesterday_maxPressureMB))
+    logger.debug("yesterday_minPressureInHg: %s ; yesterday_minPressureMB: %s" %
+                 (yesterday_minPressureInHg, yesterday_minPressureMB))
+    yesterday_maxWindMPH = str(data['maxwspdi'])
+    yesterday_maxWindKPH = str(data['maxwspdm'])
+    yesterday_minWindMPH = str(data['minwspdi'])
+    yesterday_minWindKPH = str(data['minwspdm'])
+    logger.debug("yesterday_maxWindMPH: %s ; yesterday_maxWindKPH: %s" %
+                 (yesterday_maxWindMPH, yesterday_maxWindMPH))
+    logger.debug("yesterday_minWindMPH: %s ; yesterday_minWindKPH: %s" %
+                 (yesterday_minWindMPH, yesterday_minWindKPH))
+    yesterday_maxVisibilityMI = str(data['maxvisi'])
+    yesterday_maxVisibilityKM = str(data['maxvism'])
+    yesterday_minVisibilityMI = str(data['minvisi'])
+    yesterday_minVisibilityKM = str(data['minvism'])
+    logger.debug("yesterday_maxVisibilityMI: %s ; yesterday_maxVisibilityKM: %s" %
+                 (yesterday_maxVisibilityMI, yesterday_maxVisibilityKM))
+    logger.debug("yesterday_minVisibilityMI: %s ; yesterday_minVisibilityKM: %s" %
+                 (yesterday_minVisibilityMI, yesterday_minVisibilityKM))
+    yesterday_precipMM = str(data['precipm'])
+    yesterday_precipIN = str(data['precipi'])
+    logger.debug("yesterday_precipMM: %s ; yesterday_precipIN: %s" %
+                 (yesterday_precipMM, yesterday_precipIN))
+
+    if yesterday_precipMM == "T":
+        yesterday_precipdata = False
+    else:
+        yesterday_precipdata = True
+    logger.debug("yesterday_precipdata: %s" % yesterday_precipdata)
+
+print(Fore.YELLOW + "Minimum Temperature: " + Fore.CYAN + yesterday_minTempF

```

Comment:
You don't have to show all of this data, and I'd suggest cutting a lot of it out.

I'd say just showing the high/low temperature, and the conditions. People will go into the detailed view if they want more info.

I'd suggest trying to get the weather that it was in the morning, afternoon, and evening, and you can do this in an advanced way (I can do it if you can't):
* Have a for loop that seeks out using a string find if "9:" and "AM" are in the hourly time string, "12:" and "PM", and "9:" and "PM" for the morning, afternoon, and evening
* If the string is found, get the data, and somehow flag that the data has been gotten, so that if 9:00 AM and 9:30 AM appear you're only getting 9:00 AM data.

This data parsing of course should be done before weather info is displayed.